### DISCUSSION TITLE
Plugin: Newgrounds x Kaboom

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#179] [Show and tell]
I have dedicated myself to creating a newgrounds plugin for kaboom, I love newgrounds and also kaboom! Maybe then extend the plugin to get popups and so on, but hey, I hope someone will find it useful

I explain how to use it in the readme: https://github.com/lajbel/newgrounds-boom

Hey, if you make a game for Newgrounds dot com and use this plugin, show it to me, I'll add it as a demo preview in the repository!

**Comment by slmjkdbtl** [comment]
This will be super useful, thanks!

**Reply by lajbel** [reply]
It will be easier to distribute games made in kaboom on web gaming platforms, maybe make some plugin for gamejolt api

**Reply by slmjkdbtl** [reply]
Yes! I'll look into it. Also maybe some itch.io integration?

**Reply by lajbel** [reply]
Itch.io does not have integration with trophies or scoreboards, I think the maximum is a system to upload the games without entering the page (?), even then I would not see it necessary

**Reply by slmjkdbtl** [reply]
Oh right they do not have any 😳
